src/test/java/io/confluent/examples/streams/UserCountsPerRegionLambdaIntegrationTest.java

```diff
@@ -116,29 +111,33 @@ public void shouldCountUsersPerRegion() {
 
     final TopologyTestDriver topologyTestDriver = new TopologyTestDriver(builder.build(), streamsConfiguration);
 
-    //
-    // Step 2: Publish user-region information.
-    //
-
-    IntegrationTestUtils.produceKeyValuesSynchronously(
-      inputTopic,
-      userRegionRecords,
-      topologyTestDriver,
-      new StringSerializer(),
-      new StringSerializer()
-    );
-
-    //
-    // Step 3: Verify the application's output data.
-    //
-
-    final Map<String, Long> actualClicksPerRegion = IntegrationTestUtils.drainTableOutput(
-      outputTopic,
-      topologyTestDriver,
-      new StringDeserializer(),
-      new LongDeserializer()
-    );
-    assertThat(actualClicksPerRegion).isEqualTo(expectedUsersPerRegion);
+    try {
+      //
+      // Step 2: Publish user-region information.
+      //
+
```

Comment:
Any specific reason for this spacing in last version? Seems unnecessary.

Reply:
probably just to create more visual gap to enhance readability.

Reply:
Note, the spacing didn't change in this PR. But I can remove the blank line if it bothers you.

```suggestion
```